src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import duke.command.AddDeadlineCommand;
+import duke.command.AddEventCommand;
+import duke.command.AddToDoCommand;
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.DisplayCommand;
+import duke.command.FindCommand;
+import duke.command.MarkCommand;
+import duke.command.UnMarkCommand;
+import duke.exceptions.DukeException;
+
+public class ParserTest {
+
+    @Test
+    public void testParse_validCommands() {
```

Comment:
Should the method names be testParseValidCommands() instead similar to DateParserServiceTest.java instead?